src/utils/BitlyWorker.js

```diff
@@ -1,36 +1,30 @@
-const BitlyClient = require('bitly').BitlyClient;
-const bitly = new BitlyClient(process.env.REACT_APP_BITLY_KEY);
-
 export async function generateLink(longLink) {
-  return new Promise((resolve) => {
-    bitly
-      .shorten(longLink)
-      .then(function (result) {
-        resolve({ link: result.link, errorNeeded: false });
-      })
-      .catch(function (error) {
-        if (process.env.REACT_APP_BITLY_KEY == null) {
-          console.error(
-            'Error: REACT_APP_BITLY_KEY needs to be set as an environment variable in order to share FSH Online links.'
-          );
-          resolve({ link: undefined, errorNeeded: true });
-        } else {
-          resolve({ link: undefined, errorNeeded: true });
-        }
-      });
-  });
+  return fetch('https://api-ssl.bitly.com/v4/shorten', {
```

Comment:
Just curious -- was the `bitly` library causing issues? Not maintained? Or did you just decide that bitly is easy enough that we don't need another dependency just for bitly?

Reply:
All of the above actually. The `bitly` library was using an old version of axios that was reporting an issue in `npm audit`. When I looked at the library, it hadn't been updated in a while and the calls were easy enough when I looked at the documentation, so I figured it was worth updating to remove the dependency.